packages/neuron-wallet/src/channel.ts

```diff
@@ -243,5 +248,19 @@ const listenToChannel = () => {
 export const setLanguage = (win: Electron.BrowserWindow, lng: string) => {
   win.webContents.send(Channel.SetLanguage, lng)
 }
+export const sendTransactions = (win: Electron.BrowserWindow, pageNo: number, pageSize: number) => {
```

Comment:
The name `sendTransactions` gives us a impression that this is to send transactions to the node/blockchain.

Reply:
renamed to `sendTransactionHistory`